src/test/java/com/wikia/webdriver/testcases/infoboxbuilder/Sidebar.java

```diff
@@ -115,9 +115,9 @@ public Sidebar changeHeaderCollapsibilityState() {
     String chevronContent = getPseudoElementValue(infoboxHeader, ":after", "content");
 
     if (sidebarCheckbox.isSelected()) {
-      Assertion.assertFalse(chevronContent.isEmpty());
+      Assertion.assertFalse(chevronContent.equals("none"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "none" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)